### PR TITLE
build/artik053s: add st_things config in artik053s/artik055s

### PR DESCRIPTION
**Pull request by KimDongEon**
You can build and test with st_things setting in artik053s/artik055s

### CODE REVIEW

File: build/configs/artik053s/st_things/Make.defs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+############################################################################
+# configs/artik053s/nettest/Make.defs
```
Comment (sunghan-chang) [inline build/configs/artik053s/st_things/Make.defs]: check path

File: build/configs/artik053s/st_things/defconfig [hunk 2]
```diff
@@ -0,0 +1,1229 @@
...
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_DEVNAME="/dev/smart0p10"
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_MOUNTPOINT="/sss"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS=y
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_DEVNAME="/dev/mtdblock9"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_MOUNTPOINT="/rom"
+
+#
+# Kernel Features
+#
+CONFIG_DISABLE_OS_API=y
```
Comment (sunghan-chang) [inline build/configs/artik053s/st_things/defconfig]: please disable this.

File: build/configs/artik053s/st_things/defconfig [hunk 3]
```diff
@@ -0,0 +1,1229 @@
...
+# CONFIG_DISABLE_REALLOC_NEIGHBOR_EXTENSION is not set
+# CONFIG_MM_SMALL is not set
+CONFIG_MM_REGIONS=1
+# CONFIG_ARCH_HAVE_HEAPx is not set
+# CONFIG_GRAN is not set
+
+#
+# Power Management
+#
+CONFIG_PM=y
```
Comment (sunghan-chang) [inline build/configs/artik053s/st_things/defconfig]: Let's disable.

File: build/configs/artik053s/st_things/defconfig [hunk 4]
```diff
@@ -0,0 +1,1229 @@
...
+#
+# Exclude individual procfs entries
+#
+# CONFIG_FS_PROCFS_EXCLUDE_PROCESS is not set
+# CONFIG_FS_PROCFS_EXCLUDE_UPTIME is not set
+# CONFIG_FS_PROCFS_EXCLUDE_VERSION is not set
+# CONFIG_FS_PROCFS_EXCLUDE_MTD is not set
+# CONFIG_FS_PROCFS_EXCLUDE_PARTITIONS is not set
+# CONFIG_FS_PROCFS_EXCLUDE_SMARTFS is not set
+# CONFIG_FS_PROCFS_EXCLUDE_POWER is not set
```
Comment (sunghan-chang) [inline build/configs/artik053s/st_things/defconfig]: If you disable pm, this will be disappered.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik053s/st_things/defconfig (hunk 3 above)
Comment (KimDongEon) [reply build/configs/artik053s/st_things/defconfig]: modify

File: build/configs/artik053s/st_things/defconfig (hunk 2 above)
Comment (KimDongEon) [reply build/configs/artik053s/st_things/defconfig]: modify

File: build/configs/artik053s/st_things/Make.defs (hunk 1 above)
Comment (KimDongEon) [reply build/configs/artik053s/st_things/Make.defs]: modify

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik053s/st_things/Make.defs [hunk 5]
```diff
@@ -0,0 +1,156 @@
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
```
Comment (jeongchanKim) [inline build/configs/artik053s/st_things/Make.defs]: check copyright year

Comment (KimDongEon) [reply build/configs/artik053s/st_things/Make.defs]: modify

**Comment by seinfra** [comment]
Code Rule Check OK.

File: build/configs/artik053s/st_things/defconfig [hunk 6]
```diff
@@ -0,0 +1,1229 @@
...
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_DEVNAME="/dev/smart0p10"
+CONFIG_ARTIK05X_AUTOMOUNT_SSSRW_MOUNTPOINT="/sss"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS=y
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_DEVNAME="/dev/mtdblock9"
+CONFIG_ARTIK05X_AUTOMOUNT_ROMFS_MOUNTPOINT="/rom"
+
+#
+# Kernel Features
+#
+# CONFIG_DISABLE_OS_API in not set
```
Comment (sunghan-chang) [inline build/configs/artik053s/st_things/defconfig]: Don't modify config manually. Update this file through menuconfig.

Comment (KimDongEon) [reply build/configs/artik053s/st_things/defconfig]: I have reset it in menuconfig.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**